### PR TITLE
feat: updates and new rules

### LINKED ISSUE #4241: Error in proc_creation_win_tscon_rdp_redirect.yml?
Hello everyone,

when looking into the [proc_creation_win_tscon_rdp_redirect](https://github.com/SigmaHQ/sigma/blob/master/rules/windows/process_creation/proc_creation_win_tscon_rdp_redirect.yml) rule, I noticed a difference in the command line field compared to the referenced sources.

- the rule checks if the command line contains ` /dest:rdp-tcp:`.
- the referenced sources ([1](http://www.korznikov.com/2017/03/0-day-or-feature-privilege-escalation.html), [2](https://doublepulsar.com/rdp-hijacking-how-to-hijack-rds-and-remoteapp-sessions-transparently-to-move-through-an-da2a1e73a5f6)) contain the following command line: `sc create sesshijack binpath= "cmd.exe /k tscon 1 /dest:rdp-tcp#55"`
- the difference is in the `:`/`#` charachter right after `rdp-tcp`.

Executing the given example command lines results in an event that would not trigger the rule.
Does the rule work as intended and tries to detect something different than referenced in the sources or does the rule contain a typo?

Info:
- The [tscon documentation](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/tscon) does not give any hints about the given session name with `/dest`
- This [RDP Hijacking example by red canary](https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1563.002/T1563.002.md) also shows `rdp-tcp#55`

### PR DESCRIPTION
**Pull request by nasbench**
### Summary of the Pull Request

This PR adds new rules and updates some old ones

### Detailed Description of the Pull Request / Additional Comments

- Adds new rules related to LiveKD and KD
- Adds new emerging threat related to Devil Bait
- Adds new Wscript / Certutil rules
- Update old rules logic
- Move some rules to TH folder (WIP for future PRs)

### Example Log Event

N/A

### Fixed Issues

Closes #4241 

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

File: rules/windows/process_creation/proc_creation_win_rundll32_process_dump_via_comsvcs.yml [hunk 1]
```diff
@@ -29,18 +30,27 @@ detection:
...
-    selection_cli:
+    selection_cli_1:
         CommandLine|contains|all:
             - 'comsvcs'
             - 'full'
         CommandLine|contains:
             - '24 '
             - '#24'
-            - '#+24'
+            - '#+'
```
[the remark this answers is not in the document]
Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_rundll32_process_dump_via_comsvcs.yml:40]: You can also write it as`#+0000^24` as its shown in the new ref. Removing it offers more coverage for future versions. As no ordinal call should be called with `#+` as its a high sign of some obfuscation.

[the remark this answers is not in the document]
Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_rundll32_process_dump_via_comsvcs.yml:40]: Also you could argue as you said "full" and "comsvcs" are enough. We could try it and see.

[the remark this answers is not in the document]
Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_rundll32_process_dump_via_comsvcs.yml:40]: Also Just noticed something I need to change :)

[the remark this answers is not in the document]
Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_rundll32_process_dump_via_comsvcs.yml:39]: Made some update to the rules to increase coverage for this and similar. Check this commit https://github.com/SigmaHQ/sigma/pull/4242/commits/84d849430dd9778296c6dfea6e7744095b4d80b0

Sorry for changes after review 🙈

[the remark this answers is not in the document]
Comment (phantinuss) [reply rules/windows/process_creation/proc_creation_win_rundll32_process_dump_via_comsvcs.yml:39]: I know, but as this is specific rule and the `#+` obfuscation would be its own generic rule, imo.

[the remark this answers is not in the document]
Comment (phantinuss) [reply rules/windows/process_creation/proc_creation_win_rundll32_process_dump_via_comsvcs.yml:39]: But I'm fine either way.

**Review by phantinuss — APPROVED** [review]

**Pull request merged**